### PR TITLE
Update readme file to meet standards

### PR DESCRIPTION
**Pull request by jpmaHub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,490 +1,96 @@
...
-    }
-  ]
-}
-```
+Then we have an automated four step deployment process, which runs in CircleCI.
 
-### Get payment agreement information for housing residents
+1. Automated tests (xUnit) are run to ensure the release is of good quality.
+2. The app is deployed to staging automatically, where we check our latest changes work well.
+3. We manually confirm a production deployment in the CircleCI workflow once we're happy with our changes in staging.
```
Comment (rebeccafitzsimmons1) [inline README.md]: just thinking about number 5 in the Documentation Standards - 'When do I release to production?'.  Was it agreed in Friday's retro that we would deploy to prod per each feature/config merged into central? If so, would it be appropriate to mention that here?

File: README.md [hunk 2]
```diff
@@ -1,490 +1,96 @@
...
+4. The app is deployed to production.
```
Comment (rebeccafitzsimmons1) [inline README.md]: Does it need to be mentioned where staging and production are hosted (i.e. AWS)?

File: README.md [hunk 3]
```diff
@@ -1,490 +1,96 @@
...
-     }
-   ]
- }
+- **Rashmi Shetty**, Development Manager at London Borough of Hackney (rashmi.shetty@hackney.gov.uk)
+- **Vlad Atamanyuk**, Junior Developer at London Borough of Hackney (vladyslav.atamanyuk@hackney.gov.uk)
+- **Jeff Pinkham**, Engineer at [Made Tech][made-tech] (jeff@madetech.com)
+- **Mark Rosel**, Lead Engineer at [Made Tech][made-tech] (mark.rosel@madetech.com)
+- **Steven Leighton**, Engineer at [Made Tech][made-tech] (steven@madetech.com)
+- **Cormac Brady**, Engineer at [Made Tech][made-tech] (cormac@madetech.com)
+- **Elena Vilimaite**, Engineer at [Made Tech][made-tech] (elena@madetech.com)
```
Comment (rebeccafitzsimmons1) [inline README.md]: are these the correct active maintainers?

Comment (jpmaHub) [reply README.md]: I am not sure. need to double check with @zuren

File: README.md [hunk 4]
```diff
@@ -1,490 +1,90 @@
...
-      ...etc...
-    }
-  ]
-}
-```
-### Get account and address for housing residents
+To serve the application, run it using your IDE of choice. We've Visual Studio CE on Mac.
 
-Returns a list of account and address information
+The test suite depends on a local version of Universal Housing, to simulate the legacy data source. To bring this up, run the following. You can then run the tests using the test explorer in your IDE.
```
Comment (zuren) [inline README.md]: This documentation isn't quite right, as the docker command below is for running the application, not for running the Universal Housing Simulator. The simulator is documented further down in this file, so should this paragraph be about the command beneath it?

File: README.md [hunk 5]
```diff
@@ -1,490 +1,90 @@
...
-  "beginDate": "2017-10-18T08:00:00Z",
-  "endDate": "2017-10-18T12:00:00Z",
-}
-```
-
-### Book an appointment for a Work Order
+## Development practices
 
-Creates the appointment in DRS and returns the booked appointment
+We employ a variant of Clean Architecture, borrowing from [Made Tech Flavoured Clean Architecture][mt-ca] and Made Tech's [.NET Clean Architecture library][dotnet-ca].
```
[the remark this answers is not in the document]
Comment (jpmaHub) [reply README.md]: is it ok if i remove the `Development practices` section?

[the remark this answers is not in the document]
Comment (zuren) [reply README.md]: Probably best to remove it completely IMO - hard to describe the development practices.

**Pull request merged**